From c628099732233eece07e6b266a38aac4e8236d45 Mon Sep 17 00:00:00 2001
From: Tycho Andersen <tycho.andersen@canonical.com>
Date: Mon, 13 Apr 2015 18:07:03 +0000
Subject: fix dead code

We've already checked that c is not null above, so the false branch can never
be taken here.

Reported-by: Coverity
Signed-off-by: Tycho Andersen <tycho.andersen@canonical.com>
Acked-by: Serge E. Hallyn <serge.hallyn@ubuntu.com>
---
 src/lxc/lxccontainer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lxc/lxccontainer.c b/src/lxc/lxccontainer.c
index 5ec1eec..70a838d 100644
--- a/src/lxc/lxccontainer.c
+++ b/src/lxc/lxccontainer.c
@@ -1528,7 +1528,7 @@ static bool lxcapi_createl(struct lxc_container *c, const char *t,
 		return false;
 
 	struct lxc_conf *old = current_config;
-	current_config = c ? c->lxc_conf : NULL;
+	current_config = c->lxc_conf;
 
 	/*
 	 * since we're going to wait for create to finish, I don't think we
